integration/presto/src/main/java/org/apache/carbondata/presto/PrestoFilterUtil.java

```diff
@@ -69,6 +72,13 @@
 
   private static Map<Integer, Expression> filterMap = new HashMap<>();
 
+  private final static String HIVE_DEFAULT_DYNAMIC_PARTITION = "__HIVE_DEFAULT_PARTITION__";
+  private final static String PARTITION_VALUE_WILDCARD = "";
```

Comment:
Why need to add "wildcard" for partition filter?

Reply:
In hive PARTITION_VALUE_WILDCARD is used to select all the partitions from the table when we don't have any filter conditions on the partition column in our query, but in our code we are already quering all the partitions for the above case. So we can remove this part.